lib/presentation/feature/news/widgets/top_anime_item_widget.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../models/anime_news_item_model.dart';
+
+class TopAnimeItemWidget extends ConsumerWidget {
+  const TopAnimeItemWidget({
+    required this.animeNewsItemModel,
+    Key? key,
+  }) : super(key: key);
+  final AnimeNewsItemModel animeNewsItemModel;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return GridTile(
+      footer: Material(
+        color: Colors.transparent,
+        shape: const RoundedRectangleBorder(
+          borderRadius: BorderRadius.vertical(bottom: Radius.circular(4)),
+        ),
+        clipBehavior: Clip.antiAlias,
+        child: GridTileBar(
+          backgroundColor: Colors.black45,
+          title: _GridTitleText(animeNewsItemModel.titleEn),
+          subtitle: _GridTitleText(animeNewsItemModel.titleJp),
+        ),
+      ),
+      child: Material(
+        color: Colors.black12,
```

Comment:
We can try to make an app_colors, to have a palette example in the template too

Reply:
Done, I converted the whole app to use colors from the `colors.dart` (as if defined in Figma) file rather than the built-in Colors class.